code/common/pubsub.q

```diff
@@ -174,9 +174,15 @@ init:{[t]
 
 // Initialise subscription request on startup
 .ds.subreq:(`u#`$())!`int$();
+
 // Striping function which stores the mappings for any symbols that it has already computed and 
 // for subsequent requests for that symbol, it looks them up
-.ds.stripe:{[input;skey]
+.ds.stripe:{[input]
+  // Define number of segments and match each segid to a section
```

Comment:
what is a "section" and how is it different to a "segment"?

Reply:
I don't know why I worded it like this. I'll get that changed.
```suggestion
  // Define number of segments and match to each segid
```